extendr-api/src/prelude.rs

```diff
@@ -69,8 +69,6 @@ pub use super::iter::StrIter;
 
 pub use std::convert::{TryFrom, TryInto};
 
-pub use std::ops::Index;
```

Comment:
I am surprised we don't use it. Shouldn't vector types like `Integers` have it implemented?

Reply:
I found out that what we really implement is a `Slice` trait somehow.. And that's what we use. So for `Integers`, would probably implement that, via the wrapped `Robj`.